package.json

```diff
@@ -12,6 +12,8 @@
     "format": "npm run format:md && npm run format:js",
     "format:md": "prettier --parser markdown --write '*.md'",
     "format:js": "prettier --write '**/modules/**/*.js' '**/demo/**/*.js'",
+    "format:ts": "prettier --parser typescript --write '*.ts'",
+    "fix:eslint": "eslint --fix --format=node_modules/eslint-formatter-pretty",
```

Comment:
Better extract this into a run-script while also adding TypeScript formatting through prettier.